src/modules/SdnDiag.Health.psm1

```diff
@@ -2760,6 +2761,37 @@ function Test-SdnConfigurationState {
     }
 }
 
+function Test-SdnNetworkControllerNodeRestInterface {
+    <#
+        .SYNOPSIS
+            Validates that a Network Adapter on the Network Controller node exists that matches the RestInterface name.
+    #>
+
+    [CmdletBinding()]
+    param()
+
+    Confirm-IsNetworkController
+    if ($Global:SdnDiagnostics.EnvironmentInfo.ClusterConfigType -ieq 'FailoverCluster') {
```

Comment:
Isnt this too punishing? 
We could just silently pass it, or we could have a field saying skipped/no applicable and let the execution continue.

Reply:
Shouldn't be too punishing as would require the user to directly invoke, which will just terminate and not process further. The test itself is only included in the Debug-NetworkController under the SF cluster type.